openrelik_worker_common/mount_utils.py

```diff
@@ -15,28 +15,37 @@
 import json
 import logging
 import os
+import pathlib
+import redis
 import shutil
+import socket
 import subprocess
+import time
 
 from uuid import uuid4
 
-
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 
 class BlockDevice:
-    """BlockDevice provides functionality to map image files to block devices
-    and mount them.
+    """BlockDevice provides functionality to map a disk image file to block devices
+    and mount them. The default minimum partition size that gets mounted is 100MB.
+    NOTE: If running in a container the container needs:
+    * to be privileged (due to mounting)
+    * needs access to /dev/loop* and /dev/nbd* devices
 
     Usage:
         bd = BlockDevice('/folder/path_to_disk_image.dd')
+        bd.setup()
         mountpoints = bd.mount()
         # Do the things you need to do :)
         bd.umount()
     """
 
     MIN_PARTITION_SIZE_BYTES = 100 * 1024 * 1024  # 100 MB
+    MAX_NBD_DEVICES = 10
+    LOCK_TIMEOUT_SECONDS = 6 * 60 * 60  # 6 hours
```

Comment:
```suggestion
    LOCK_TIMEOUT_SECONDS = 21600  # 6 hours
```

Reply:
Had to think about this a while. To me the way I do it is easier to read and easier to change........ at least in my brain.....

Reply:
I figured, but I don't understand what the 6 or 60 come from